tutorials/hello-kubernetes/README.md

```diff
@@ -326,7 +326,7 @@ Get the API call logs of the Python app:
 
 <!-- STEP
 expected_stdout_lines:
-  - 'level=info msg="HTTP API Called: POST /neworder"'
+  - 'level=info msg="HTTP API Called: POST /neworder'
```

Comment:
extra changes made it in.  these ok?

Reply:
Also could you please add a sentence or two about what this is, for posterity :)

Reply:
Yes!